File: solutions/1482-how-many-numbers-are-smaller-than-the-current-number/solution.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int len = nums.length;
        int[] output = new int[len];

        for(int i=0;i<len;i++){
            int small = 0;
            for(int j=0;j<len;j++){
                if(nums[i] > nums[j]){
                    small++;
                }
            }
            output[i] = small;
        }
        return output;
    }
}

File: solutions/1482-how-many-numbers-are-smaller-than-the-current-number/solution.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        op =[]
        for x in nums:
            count =0
            for y in nums:
                if x > y:
                    count+=1
            op.append(count)
        return op
        
